README.md

```diff
@@ -17,7 +17,11 @@
 
 
 ## Live Demo
-- at  https://angry-borg-c6fc5f.netlify.app/
+- at [Live Demo](https://angry-borg-c6fc5f.netlify.app/)
+
+## Video Link
+
+at [Video Link](https://drive.google.com/file/d/16wc568N9VZYNf3OdTY__uGTyeUs9UM0V/view?usp=sharing)
 
 ## Getting Started
 
```

Comment:
Your _ReadMe_ needs some improvements so as to make it appear more professional. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this capstone project,  which will likely feature on your portfolio. Therefore, kindly ensure to improve upon these aspects of your ReadMe:
- [ ]  _Project description_, _how to set up_, _run and test this project_ **should be detailed, clear, and well** organized.
- [ ] The _code_ aspects in each procedure should be quoted with **backticks**.
- [ ] Include necessary screenshots, project demo links where applicable so as to give a birds-eye view of your project
- [ ] All _typos_ **should be corrected**. You can crosscheck by installing the [Grammarly plugin](https://www.grammarly.com/)